test_setup/patch-console.js

```diff
@@ -10,6 +10,8 @@ global.console.trace = () => {
   throw new Error("Please don't call console.trace in the codebase");
 };
 
-global.console.log = () => {
+const log = global.console.log;
+global.console.log = message => {
+  log(message);
```

Comment:
I added this here so I can add console logs temporarily when debugging but still have this fail.